opencti-platform/opencti-graphql/src/schema/general.js

```diff
@@ -88,6 +88,7 @@ export const ABSTRACT_TYPES = [
   ABSTRACT_STIX_CORE_OBJECT,
   ABSTRACT_STIX_DOMAIN_OBJECT,
   ENTITY_TYPE_CONTAINER,
+  ENTITY_TYPE_CONTAINER_CASE,
```

Comment:
This is a small change that actually hides a lot of cleanup to do!
Case is seemingly abstract, but it defines stix-converter, register in the schema... like a regular type. For another PR.